From 028f9e47dc652c1b5d0e0398ffcf68e8c63c1b52 Mon Sep 17 00:00:00 2001
From: etjson <etjson@gmail.com>
Date: Wed, 19 Apr 2023 14:29:32 +0800
Subject: [PATCH 02/12] server/server.vala

---
 server/server.vala | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/server/server.vala b/server/server.vala
index b758be0..f06c894 100644
--- a/server/server.vala
+++ b/server/server.vala
@@ -1,7 +1,7 @@
 namespace Frida.Server {
 	private static Application application;
 
-	private const string DEFAULT_DIRECTORY = "re.frida.server";
+	private static string? DEFAULT_DIRECTORY = null;
 	private static bool output_version = false;
 	private static string? listen_address = null;
 	private static string? certpath = null;
@@ -50,6 +50,7 @@ namespace Frida.Server {
 	};
 
 	private static int main (string[] args) {
+		DEFAULT_DIRECTORY = GLib.Uuid.string_random();
 		Environment.init ();
 
 #if DARWIN
-- 
2.35.3.windows.1

