From 9ff652c1a6bb042fbb6eac2d2678700a78286b71 Mon Sep 17 00:00:00 2001
From: etjson <etjson@gmail.com>
Date: Wed, 19 Apr 2023 14:31:16 +0800
Subject: [PATCH 03/12] src/agent-container.vala

---
 src/agent-container.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/agent-container.vala b/src/agent-container.vala
index a8db6b2..afe514c 100644
--- a/src/agent-container.vala
+++ b/src/agent-container.vala
@@ -25,7 +25,7 @@ namespace Frida {
 			assert (container.module != null);
 
 			void * main_func_symbol;
-			var main_func_found = container.module.symbol ("frida_agent_main", out main_func_symbol);
+			var main_func_found = container.module.symbol ("main", out main_func_symbol);
 			assert (main_func_found);
 			container.main_impl = (AgentMainFunc) main_func_symbol;
 
-- 
2.35.3.windows.1

